osidb/api_views.py

```diff
@@ -360,6 +361,44 @@ def whoami(request: Request) -> Response:
     return Response(UserSerializer(request.user).data)
 
 
+@include_meta_attr_extend_schema_view
+@include_exclude_fields_extend_schema_view
+@extend_schema_view(
+    create=extend_schema(
+        request=FlawCommentSerializer,
```

Comment:
I think that this is not necessary as `serializer_class` defines the default serializer and is equal.

Reply:
Resolved in commit 76a8581033d95a8a77a598c876457e2b162a158f